### ISSUE TITLE
export issue

### ISSUE THREAD

**Issue by gaful** [lazymofo/datagrid#63]
Hello, 
First, thanks for this great code. It is very usefull. 
I am not sure if this is an issue, or me misusing the code. 

I want to use the code to print different tables that are the same. So I have added a small piece of code.

<form method='post'  action='<?php echo $_SERVER['PHP_SELF']  ?>' >
	<select name='dbtable' id='dbtable'  onChange='this.form.submit()'>
		<option value='table1'>table1</option>
		<option value='table2'>table2</option>
	</select>
</form>

my request looks like this : 
$lm->grid_sql = "select t.a, t.b
	from $varTable t 
	where (coalesce(t.a, '') like :_search 
	or    coalesce(t.b, '') like :_search )
	order by t.client desc
	"

Basically it allows the user to switch easily from one table to another. 
It does work great for udpating data in the browser. But for the export I always have the same export of the first table I have viewed and the export never change. 

Any idea what I am missing ? 

Thank you
G
